lib/stripe/payment_intent_validator.rb

```diff
@@ -14,6 +16,16 @@ def call(payment_intent_id, stripe_account_id)
 
     private
 
+    def payment_intent(payment)
```

Comment:
I'd keep payment_intent_id as it is not the full payment intent object we are handling here.